### PR TITLE
SAAP-198 Azure cloud provider Docs Update

### PR DESCRIPTION
**Pull request by TehreemNisa**
Updating Azure cloud provider docs

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-138-8ec2ecca`

File: content/sre/cloud-provider/azure.md [hunk 1]
```diff
@@ -14,17 +14,12 @@ An azure subscription is needed to create and manage cluster on azure. The follo
...
+  | Public IP addresses | 5 |
+  | Private IP Addresses | 7 |
+  | Network Interfaces | 6 |
+  | Network Load Balancers   | 3 |
+  | Virtual Networks | 1 |
   | StandardStorageSnapshots | 10000 (depends on how many disks are used) and backup duration |
-  | Network Security Groups | 6 |
-  | Premium Storage Managed Disks | 100 (depends on how many disks are used) |
-  | Network Interfaces | 30 |
+  | Machine Specifications | 6 machines of 8x32x120G |
```
Comment (usamaahmadkhan) [inline content/sre/cloud-provider/azure.md:25]: Add this here as well:

`| Region | Region will be identified by the customer |`

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: content/sre/cloud-provider/azure.md (hunk 1 above)
Comment (TehreemNisa) [reply content/sre/cloud-provider/azure.md:25]: done

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-138-2a4d3ab2`

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.84`

**Pull request merged**